main.py

```diff
@@ -119,14 +120,25 @@ def setupLogging(verbose_notifs, notifier):
 
 
 def cleanupChromeProcesses():
-    # Use psutil to find and terminate Chrome processes
-    for process in psutil.process_iter(["pid", "name"]):
-        if process.info["name"] == "chrome.exe":
-            try:
-                psutil.Process(process.info["pid"]).terminate()
-            except (psutil.NoSuchProcess, psutil.AccessDenied, psutil.ZombieProcess):
-                pass
-
+    # Get the current user's PID
+    current_pid = os.getpid()
+
+    # Get the list of all processes
+    all_processes = psutil.process_iter()

```

Comment:
I agree with this, this should be in terminate_children. The minor speed loss is not consequential enough to matter here.

Reply:
I agree with this, this should be in terminate_children. The minor speed loss is not consequential enough to matter here.

Edit: Looking back at this again, it might be fine given that terminate_children is declared within the current function.